nmdc_runtime/api/endpoints/nmdcschema.py

```diff
@@ -35,6 +36,41 @@ def get_nmdc_schema_version():
     return version("nmdc_schema")
 
 
+@router.get("/nmdcschema/collection_stats")
+def get_nmdc_database_collection_stats(
+    mdb: MongoDatabase = Depends(get_mongo_db),
+):
+    """
+    Get stats for nmdc:Database MongoDB collections
+
+    Field reference: <https://www.mongodb.com/docs/manual/reference/command/collStats/#std-label-collStats-output>.
+    """
+    present_collection_names = set(nmdc_database_collection_names()) & set(
+        mdb.list_collection_names()
+    )
+    stats = []
+    for n in present_collection_names:
+        for doc in mdb[n].aggregate(
+            [
+                {"$collStats": {"storageStats": {}}},
+                {
+                    "$project": {
+                        "ns": 1,
```

Comment:
I was going to say that I wasn't familiar with the label `ns`, but I see that it is documented on the web page linked in the Swagger documentation for the endpoint, at: https://www.mongodb.com/docs/manual/reference/command/collStats/#mongodb-data-collStats.ns